cfme/utils/log_validator.py

```diff
@@ -37,8 +48,8 @@ class LogValidator(object):
                                   skip_patterns=['PARTICULAR_ERROR'],
                                   failure_patterns=['.*ERROR.*'],
                                   matched_patterns=['PARTICULAR_INFO'])
-          evm_tail.fix_before_start()
-          evm_tail.validate_logs()
+          evm_tail.start_monitoring()
+          evm_tail.validate()       # evm_tail.validate(wait="30s")
```

Comment:
Thoughts on adding a line or two about appropriate kwargs to pass for `wait`? E.g. 
```python,
"""
It is also possible for  wait for log validation, the proper syntax for doing so is:
evm_tail.validate(wait="30s")
The appropriate strings to pass to wait are ... x,y, z
"""
```

Reply:
John thanks for this, but we try to cover simple usecase as example in doc-string mainly when use `code-block`. I think its sufficient to pass usecase of logvalidator to user and after that he/she can checks other available methods like matches etc....

Let me know if you still want this; I will enhance this `code-block`.

Reply:
That's fine with me, I would consider it optional. As it matches with the usage of `wait` elsewhere in the codebase.